### DISCUSSION TITLE
about vtkBooleanOperationPolyDataFilter()

### DISCUSSION THREAD

**Discussion by ZzZuf** [Kitware/trame#505] [General]
Hello:
        I need to fillet the model in the image below. My method is to generate a model that intersects the original model at each of the four cusps of the model, and perform a Boolean operation with the original model to obtain the difference. But I've found that using vtkBooleanOperationPolyDataFilter() directly gives me an error. In C++, I took a workaround of customizing a class that inherits from vtkBooleanOperationPolyDataFilter() and overriding the RequestData() method in it. However, in Python I found that vtkBooleanOperationPolyDataFilter() doesn't have a RequestData() method. Is there any way to fix this?
![3659ef4c61fc11354c9652188b98cea](https://github.com/Kitware/trame/assets/162105830/87e44843-a75b-4562-a656-dbef823c0d4a)
![b76b64ac99fd9eea2397d96abaed55e](https://github.com/Kitware/trame/assets/162105830/71c6521b-df28-4eef-b65e-08b2752097c9)
![4a473f505bc415fc3ac0bea7466b131](https://github.com/Kitware/trame/assets/162105830/da7ba682-2d63-4825-b0ed-7b7d7cae354e)

**Comment by jourdain** [comment]
You will have a better chance to get an answer if you ask that question on [the vtk discourse](https://discourse.vtk.org/).
